src/map/clif.h

```diff
@@ -970,7 +970,8 @@ struct clif_interface {
 	void (*skillname_ack) (int fd, struct block_list *bl);
 	void (*itemname_ack) (int fd, struct block_list *bl);
 	void (*unknownname_ack) (int fd, struct block_list *bl);
-	void (*monster_hp_bar) ( struct mob_data* md, struct map_session_data *sd );
+	void (*monster_hp_bar) ( struct mob_data *md, struct map_session_data *sd );
```

Comment:
and probably last request.
here should be ``void (*monster_hp_bar) (struct mob_data *md, struct map_session_data *sd);`` and need remove spaces in same way in next line

Reply:
@4144 fixed in last commit